demos/Ollama Chat Service/README.md

```diff
@@ -0,0 +1,95 @@
+# Ollama Chat Service Demo
+
+There are two demonstrations of the chat web-app that use Ollama as an LLM chat
+service.  The following instructions describe how to set up a local Ollama
+service with which the web-app can interact, send queries, and show the chat
+service's responses.
+
+## Ollama Service
+
+First, set up the Ollama service. Follow the directions found in the [Ollama
+Github project](https://github.com/ollama/ollama?tab=readme-ov-file).  The demo
+web-apps used the Phi-3 language model.  Follow the instructions given in
+Ollama's [Model
+Library](https://github.com/ollama/ollama?tab=readme-ov-file#model-library) to
+install and run that language model.
+
+Once the Ollama Service is installed, launched and running, proceed to the next
+steps.
+
+## Chat Web-App
+
+There are two versions of the web-app.  They appear identical within the browser
```

Comment:
It would be helpful to compare or explain the pros and cons of these two versions.

Reply:
Done.